etl-pipeline/processes/api.py

```diff
@@ -7,7 +7,8 @@
 
 class APIProcess:
     def __init__(self, *args):
-        self.db_manager = DBManager()
+        host = os.environ.get("POSTGRES_READ_HOST")
+        self.db_manager = DBManager(host=host)
```

Comment:
Thoughts on falling back here to POSTGRES_HOST? Otherwise, let's set it in the other config files as well.

Reply:
This already happens by default in the DBManager!